package.json

```diff
@@ -68,9 +68,11 @@
     "chai-string": "^1.3.0",
     "chalk": "^2.1.0",
     "child_process": "^1.0.2",
+    "chunk": "0.0.2",
```

Comment:
This is like 10 lines of code. I would rather copy that code to some utils or use lodash here than some random library I'm not aware of. Security reasons.